src/src/Helpers/HTTPResponseParser.cpp

```diff
@@ -16,203 +16,193 @@
 
 # endif // if FEATURE_JSON_EVENT
 
+int parseJson = -1; // indicates whether the URI lacks a JSON flag (-1), or if present, specifies the group number (>= 0)
```

Comment:
Why a static variable for this?
Is it guaranteed the response and event will always be in expected order?